src/sdg/producten/models/managers.py

```diff
@@ -42,6 +43,14 @@ def active(self, active_on=None):
             )
         )
 
+    def active_organization(self):
+        """
+        Filter the products to only include products from active organizations.
+        """
+        return self.filter(
+            catalogus__lokale_overheid__organisatie__owms_end_date__gte=now()
```

Comment:
This queryset will not show organisations that have no (`None`) end date. And this is the majority of them all.

```suggestion
            Q(catalogus__lokale_overheid__organisatie__owms_end_date__gte=now() | catalogus__lokale_overheid__organisatie__owms_end_date__isnull=True)
```

This also applies to the other 3 locations where you added the function.